src/config/theme.ts

```diff
@@ -16,6 +16,7 @@ declare module '@mui/material/Alert' {
     low: true;
     medium: true;
     high: true;
+    gray: true;
```

Comment:
This seems good to add since there is a `grayscale` Alert in the new design library. Can you rename it to `grayscale` to match that?

Reply:
I followed up with Bobby on the design language about the introduction of this "grayscale alert" and we landed on _not_ adding it since it doesn't fit into the MUI severity paradigm https://www.figma.com/design/bsw8CX4UOq6SIdXYorZDWV?node-id=2482-931#1173966176

So suggest removing this altogether!